OsmAnd/src/net/osmand/plus/settings/backend/OsmandSettings.java

```diff
@@ -1117,6 +1117,26 @@ public SpeedConstants getProfileDefaultValue(ApplicationMode mode) {
 		}
 	}.makeProfile();
 
+	public final OsmandPreference<VolumeUnit> UNIT_OF_VOLUME = new EnumStringPreference<>(this,
+			"unit_of_volume", VolumeUnit.LITRES, VolumeUnit.values()) {
+
+		@Override
+		public VolumeUnit getProfileDefaultValue(ApplicationMode mode) {
```

Comment:
We can probably make similar to [MetricsConstants ](https://github.com/osmandapp/Osmand/blob/817faeedeaf8d7ffecdd11108d2da683c720b341/OsmAnd/src/net/osmand/plus/settings/backend/OsmandSettings.java#L1055-L1062)in DrivingRegion